happypose/toolbox/utils/download.py

```diff
@@ -173,27 +180,135 @@ def main():
         )
         for result_id in (PBR_INFERENCE_ID, SYNT_REAL_INFERENCE_ID, SYNT_REAL_ICP_INFERENCE_ID,
                           SYNT_REAL_4VIEWS_INFERENCE_ID, SYNT_REAL_8VIEWS_INFERENCE_ID):
-            download(f'cosypose/results/{result_id}', LOCAL_DATA_DIR / 'results')
+            download(f'{DOWNLOAD_URL}/cosypose/results/{result_id}', LOCAL_DATA_DIR / 'results')
 
 
-def run_rclone(cmd, args, flags):
-    rclone_cmd = ['rclone', cmd] + args + flags + ['--config', str(RCLONE_CFG_PATH)]
-    logger.debug(' '.join(rclone_cmd))
-    print(rclone_cmd)
-    subprocess.run(rclone_cmd)
+#manages multiple downloads in async
+def downloads(*args):
```

Comment:
```suggestion
def downloads(*args):
    """manages multiple downloads in async"""
```
This looks more like a documentation than a comment.
In python, to document a function, the doc string is the first non captured string of the function.